modules/insticatorBidAdapter.js

```diff
@@ -17,17 +17,17 @@ export const OPTIONAL_VIDEO_PARAMS = {
   'maxduration': (value) => isInteger(value),
   'protocols': (value) => isArrayOfNums(value), // protocols values supported by Inticator, according to the OpenRTB spec
   'startdelay': (value) => isInteger(value),
-  'linearity': (value) => isInteger(value) && [1].includes(value),
-  'skip': (value) => isInteger(value) && [1, 0].includes(value),
   'skipmin': (value) => isInteger(value),
-  'skipafter': (value) => isInteger(value),
-  'sequence': (value) => isInteger(value),
+  'linearity': (value) => isInteger(value) && [1].includes(value),
   'battr': (value) => isArrayOfNums(value),
+  'skipafter': (value) => isInteger(value),
+  'skip': (value) => isInteger(value) && [1, 0].includes(value),
   'maxextended': (value) => isInteger(value),
-  'minbitrate': (value) => isInteger(value),
   'maxbitrate': (value) => isInteger(value),
-  'playbackmethod': (value) => isArrayOfNums(value),
```

Comment:
Hi @shubhamc-ins this commit is poor conduct and will not be tolerated

Please resubmit

Reply:
@patmmccann apologies, I will resubmit.